themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties

```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
 internalApps=内部アプリケーション
 thirdpartyApps=サードパーティのアプリケーション
 appResults=結果
+
+# Linked account
+authorizedProvider=認可済みプロバイダー
+authorizedProviderMessage=アカウントにリンクされた認可済みプロバイダー
+identityProvider=アイデンティティー・プロバイダー
+identityProviderMessage=アカウントと設定したアイデンティティー・プロバイダーをリンクするには
+socialLogin=ソーシャル・ログイン
+userDefined=ユーザー定義
+removeAccess=アクセスの削除
+removeAccessMessage=このアプリ・アカウントを使用する場合は、アクセス権を再度付与する必要があります。
+
+#Authenticator
+authenticatorStatusMessage=2要素認証は現在
+authenticatorFinishSetUpTitle=あなたの2要素認証
+authenticatorFinishSetUpMessage=Keycloakアカウントにサインインするたびに、2要素認証コードを入力するように求められます。
+authenticatorSubTitle=2要素認証を設定する
+authenticatorSubMessage=アカウントのセキュリティーを強化するには、利用可能な2要素認証の方式のうち少なくとも1つを有効にします。
+authenticatorMobileTitle=モバイル・オーセンティケーター
+authenticatorMobileMessage=モバイル・オーセンティケーターを使用して、2要素認証として確認コードを取得します。
+authenticatorMobileFinishSetUpMessage=オーセンティケーターはあなたの携帯電話にバインドされています。
+authenticatorActionSetup=セットアップ
+authenticatorSMSTitle=SMSコード
+authenticatorSMSMessage=Keycloakは、2要素認証として確認コードを携帯電話に送信します。
+authenticatorSMSFinishSetUpMessage=送信されるテキスト・メッセージ
+authenticatorDefaultStatus=デフォルト
+authenticatorChangePhone=電話番号の変更
+authenticatorBackupCodesTitle=バックアップ・コード
+authenticatorBackupCodesMessage=8桁のバックアップコードの入手
+authenticatorBackupCodesFinishSetUpMessage=この時点で12個のバックアップコードが生成されました。それぞれ一度だけ使用できます。
+
+#Authenticator - Mobile Authenticator setup
+authenticatorMobileSetupTitle=モバイル・オーセンティケーターのセットアップ
+smscodeIntroMessage=電話番号を入力すると、確認コードがあなたの電話に送信されます。
+mobileSetupStep1=携帯電話にオーセンティケーター・アプリケーションをインストールします。ここにリストされているアプリケーションがサポートされています。
+mobileSetupStep2=アプリケーションを開き、バーコードをスキャンしてください。
+mobileSetupStep3=アプリケーションから提供されたワンタイムコードを入力し、保存をクリックしてセットアップを終了します。
+scanBarCode=バーコードをスキャンしますか？
+enterBarCode=ワンタイムコードを入力してください
+doCopy=コピー
+doFinish=終了
+
+#Authenticator - SMS Code setup
+authenticatorSMSCodeSetupTitle=SMSコードのセットアップ
+smscodeIntroMessage=電話番号を入力すると、確認コードがあなたの電話に送信されます。
+chooseYourCountry=国を選んでください
+enterYourPhoneNumber=電話番号を入力してください
+sendVerficationCode=確認コードの送信
+enterYourVerficationCode=確認コードを入力してください
+
+#Authenticator - backup Code setup
+authenticatorBackupCodesSetupTitle=バックアップコードのセットアップ
+backupcodesIntroMessage=携帯電話にアクセスできない場合でも、バックアップコードを使用してアカウントにログインできます。どこか安全でアクセス可能な場所に保管してください。
+realmName=レルム
+doDownload=ダウンロード
+doPrint=印刷
+doCopy=コピー
+backupCodesTips-1=各バックアップコードは1回使用できます。
+backupCodesTips-2=これらのコードはこの日に生成されました：
+generateNewBackupCodes=新しいバックアップコードを生成する
+backupCodesTips-3=新しいバックアップコードを生成すると、現在のコードは機能しなくなります。
+backtoAuthenticatorPage=認証ページに戻る
+
+
+#Resources
+resources=リソース
+myResources=私のリソース
```

Comment:
@k-tamura This is duplicate message. The message was removed from English resource.
https://github.com/keycloak/keycloak/commit/f35d3a8ec9a0c404d6e92c19babc520843ead368#diff-212b66440fb033927a3a24caf41983c9L330